tests/helper_test.py

```
@@ -0,0 +1,141 @@
+
+class Node:
+    text: str
+
+
+class Color(Enum):
+    RED = 1
+
+
+class TestStdLibrary(StdLibrary):
+    @patch('fastkml.helpers.get_value')
+    def test_node_text(self, mock_get_value) -> None:
```

Ruff

Missing type annotation for function argument `mock_get_value`

(ANN001)